src/model-viewer-base.ts

```diff
@@ -295,6 +296,10 @@ export default class ModelViewerElementBase extends UpdatingElement {
     this[$scene].isDirty = true;
   }
 
+  [$needsShadowUpdate]() {
+    this[$scene].updateStaticShadow();
```

Comment:
Is the static shadow still better than standard Three.js shadows if we are updating it in real time?

Reply:
That's a good question.

I can remove all the custom code and use Three.js' built-in shadows (will fix issue 2), but in order to get that to work `auto-rotate` will have to affect the camera instead of the model.

Are you okay with that?

Reply:
Can you explain why that is the case?

Reply:
Yeah, I'll explain later.

Reply:
I was going to recommend that change anyway, since once we do #506, I would expect to get camera events describing the auto-rotation, but as it is now, the camera appears static.